cmds/core/uptime/uptime.go

```diff
@@ -11,26 +11,32 @@ package main
 import (
 	"errors"
 	"fmt"
+	"io"
 	"log"
 	"os"
 	"strings"
 	"time"
 )
 
+var (
+	errAverageFormat = errors.New("the contents of proc/loadavg we are trying to process contain less than the required 3 loadavgs")
```

Comment:
This is a bit wordy, can we just do something like
errAverageFormat = fmt.Errorf("/proc/loadavg has less than 3 fields:%v", os.ErrInvalid)
errUptimeFormat("/proc/uptime is empty: %v", os.ErrInvalid)

or some such, The os.ErrInvalid is for tests.

Reply:
We already have errUptimeFormat for test so I think os.ErrInvalid is not needed.